.webpack/webpack.dev.js

```diff
@@ -0,0 +1,63 @@
+const path = require('path');
+const webpack = require('webpack');
+// Plugins
+const { CleanWebpackPlugin } = require('clean-webpack-plugin');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+
+const ENTRY_VIEWPORT = path.join(__dirname, './../src/index.js');
+const ENTRY_EXAMPLES = path.join(__dirname, './../examples/index.js');
+const SRC_PATH = path.join(__dirname, './../src');
+const OUT_PATH = path.join(__dirname, './../dist');
+
+module.exports = {
+  entry: {
+    examples: ENTRY_EXAMPLES,
+  },
+  devtool: 'source-map',
+  output: {
+    path: OUT_PATH,
+    filename: '[name].bundle.[hash].js',
+    library: 'cornerstoneViewport',
```

Comment:
If you are changing the name, you also need to change the package.json.

Reply:
Actually just the dev server's config. Still using rollup on this one for publishing packages. Dev server's just handy for.. Developing ^_^